MAUI/Calendar/overview.md

```diff
@@ -0,0 +1,96 @@
+
+   {% multiple_column split:2 %}
+   {% column %}
+   ![Month view in .NET MAUI Calendar.](images/overview/maui-month-view.png)
+   {% endcolumn}
+   {% column %}
+   ![Year view in .NET MAUI Calendar.](images/overview/maui-year-view.png)
+   {% endcolumn}
+   {% column %}
+   ![Decade view in .NET MAUI Calendar.](images/overview/maui-decade-view.png)
+   {% endcolumn}
+   {% column %}
+   ![Century view in .NET MAUI Calendar.](images/overview/maui-century-view.png)
+   {% endcolumn}
+   {% endmultiple_column %}
+
+* **Quick navigation**: You can easily navigate to the desired date using different calendar views in the SfCalendar.
+
+* **Enable and disable built-in view switching**: Restricts or enable navigation to different calendar views through interaction.
+
+   % multiple_column split:2 %}
+   {% column %}
+   ![Month view single selection in .NET MAUI Calendar.](images/overview/maui-month-view-single-selection.png)
+   {% endcolumn}
+   {% column %}
+   ![Year view single selection in .NET MAUI Calendar.](images/overview/maui-year-view-single-selection.png)
+   {% endcolumn}
+   {% column %}
+   ![Decade view single selection in .NET MAUI Calendar.](images/overview/maui-decade-view-single-selection.png)
+   {% endcolumn}
+   {% column %}
+   ![Century view single selection in .NET MAUI Calendar.](images/overview/maui-century-view-single-selection.png)
+   {% endcolumn}
+   {% endmultiple_column %}
+
+* **Date selection**: Date selection allows you to select the Single date, Multiple dates or a Range of dates by interaction or programmatic selection.
+
+   % multiple_column split:2 %}
+   {% column %}
+   ![Single selection in .NET MAUI Calendar.](images/overview/maui-month-view-single-selection.png)
+   {% endcolumn}
+   {% column %}
+   ![Multiple selection in .NET MAUI Calendar.](images/overview/maui-multiple-selection.png)
+   {% endcolumn}
+   {% column %}
+   ![Range selection in .NET MAUI Calendar.](images/overview/maui-range-selection.png)
+   {% endcolumn}
+   {% endmultiple_column %}
+
+* **Limit date selection**: Restricts interaction and navigation beyond the specified minimum and maximum dates.
+
+   ![Limit date selection in .NET MAUI Calendar.](images/overview/maui-min-max-date.png)
+
+* **Change first day of week**: You can customize the starting day of the week. The default first day of the week is Sunday.
+
+   ![First day of week in .NET MAUI Calendar.](images/overview/maui-first-day-of-week.png)
+
+* **Selectable Day Predicate**: Disable the interaction for calendar view dates and easily prevent the selection of weekends by disabling them.
+
+   ![Selectable day predicate in .NET MAUI Calendar.](images/overview/maui-selectable-day-predicate.png)
+
+* **Highlight weekends and special dates**: Can highlight special dates and weekend dates in month view using different background color and textstyles in MAUI Calendar.
```

Comment:
In MAUI Calendar, you can highlight special dates and weekend dates in the month view using different background colors and text styles.

Reply:
Modified the content